pkg/recipes/terraform/config/providers/aws.go

```diff
@@ -43,6 +44,14 @@ const (
 	awsRegionParam    = "region"
 	awsAccessKeyParam = "access_key"
 	awsSecretKeyParam = "secret_key"
+
+	// configs for AWS IRSA
+	// Ref: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#assuming-an-iam-role-using-a-web-identity
+	awsIRSAProvider = "assume_role_with_web_identity"
+	awsRoleARN      = "role_arn"
+	sessionName     = "session_name"
+	tokenFile       = "web_identity_token_file"
+	tokenFilePath   = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
```

Comment:
Could you add a comment about this value? Is it always constant or it depends on another configuration Radius sets? A link to relevant AWS docs would be helpful as well.